app/code/Bogdank/SupportChat/view/frontend/web/js/chat-form.js

```diff
@@ -23,6 +24,11 @@ define([
             });
 
             $(this.element).on('submit.bogdank_SupportChat', $.proxy(this.saveChat, this));
+
+            console.log(customerData.get('customer-message')());
```

Comment:
@holmatro90 Forgotten debug statement. Debug code must not appear uncontrolled in the production code